lib/client.js

```diff
@@ -79,7 +79,7 @@ client.prototype.request = function(method, path, params, callback){
 
     if(["POST", "PUT"].indexOf(http_options["method"]) >= 0){
         http_options["headers"] = {
-            "Content-Type": "application/json",
+            "Content-Type": params["contentType"] ? params["contentType"] : "application/json",
```

Comment:
yeah it got me too. I could forward the email they sent me to you if you would like to read it. It was a simple ticket that I opened up with them asking if they could help. They responded by telling me the ContentType was wrong. Not sure of the reasoning behind this.